
---
moveit_msgs/RobotTrajectory tarjectory